python/src/scipp/plotting/objects.py

```diff
@@ -241,6 +251,14 @@ def __init__(self,
             if dim in self.dims:
                 self._tool_button_states[f'log_{dim}'] = scale[dim] == 'log'
 
+        if resampling_mode is None:
```

Comment:
Would it be better to make the resampling mode explicit? I can see pros/cons of each approach, but as is we are partially making implicit decisions for users based on unit.

Reply:
I agree that making this explicit would be the "correct" choice, however, I have major usability concerns. Running `da.plot()` is much more convenient than running `da.plot(resampling_mode='sum`)` and most likely we would get a huge amount of complaints about the latter.... and even if it gets used, people may just copy-and-paste, without thinking about which mode they actually need.

My hope was that the toolbar button would be explicit enough. It will not be visible for static plots or saved figures though. Maybe it can be incorporated into the labels?

Reply:
Yes it was the static plots and exported figures I was thinking about with my comment above. I think the solution here is probably the right balance between control and selected defaults for the most common use-case. Including in labels/title might be a good option to ensure that we know what resampling model is when we see plots without toolbars.

Reply:
So say we have `[K]` as the axis label, how should we indicate this?

`sum [K]` and `mean [K]` looks a bit odd?